webapi/Skills/ChatSkills/ExternalInformationSkill.cs

```diff
@@ -204,25 +205,6 @@ public class ExternalInformationSkill
 
     #region Private
 
-    /// <summary>
```

Comment:
Should we really return an empty string at line 203??

Reply:
Yes, this is intended.